src/test/java/uk/gov/hmcts/reform/pip/data/management/errorhandling/GlobalExceptionHandlerTest.java

```diff
@@ -305,4 +307,17 @@ void testHandleLocationNameValidationException() {
         assertEquals(TEST_MESSAGE, responseEntity.getBody().getMessage(),
                      ASSERTION_MESSAGE);
     }
+
+    @Test
+    void testHandleLcsuArtefactNotSupportedException() {
+        String errorMessage = "LCSU artefact type is not supported.";
+        LcsuArtefactNotSupportedException exception = new LcsuArtefactNotSupportedException(errorMessage);
+
+        ResponseEntity<ExceptionResponse> response = globalExceptionHandler.handle(exception);
+
+        assertNotNull(response, "Response should not be null");
+        assertEquals(400, response.getStatusCodeValue(), "Status code should be 400");
```

Comment:
We can re-use the assertion message defined in the variables at the top of this class. Similar for the other assertion messages